gulpfile.js

```diff
@@ -9,6 +9,7 @@ const tmplSrc = path.resolve(__dirname, './src/renderer/templates/**/*.pug');
 const tmplDest = path.resolve(__dirname, './src/renderer/angular/');
 const scssSrc = path.resolve(__dirname, './static/stylesheets/scss/main.scss');
 const scssDest = path.resolve(__dirname, './static/stylesheets/css');
+const npmcheck = require('gulp-npm-check');
```

Comment:
@sk91 can't we just use the https://www.npmjs.com/package/npm-check in a script on the package.json? We are trying to remove the dependency on gulp from the wallet and leave only webpack.  cheers

Reply:
@rodrigopavezi  you are correct, fixing